collect_app/src/main/java/org/odk/collect/android/activities/GeoTraceActivity.java

```diff
@@ -381,12 +453,10 @@ private void buildDialogs() {
     }
 
     private void startGeoTrace() {
-        RadioGroup rb = traceSettingsView.findViewById(R.id.radio_group);
-        View radioButton = rb.findViewById(rb.getCheckedRadioButtonId());
-        recordingMode = rb.indexOfChild(radioButton);
-        if (recordingMode == 0) {
+        if (recordingMode == MANUAL_RECORDING) {
             setupManualMode();
-        } else if (recordingMode == 1) {
+        }
+        if (recordingMode == AUTOMATIC_RECORDING) {
```

Comment:
These are mutually exclusive right? Can you keep the `else if`?

Reply:
Yeah, I think it got lost in one of my rebases.  It's back now.